### PR TITLE
Fix math expresion in hmong

### PR DESCRIPTION
**Pull request by crissthiandi**
Before
 
image

After

image

Only invert `.mwe-math-element` class

I include all future Hmong projects by using "*" after the "." `hmong.*/wiki/*`

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7476,6 +7476,13 @@ body
 
 ================================
 
+hmong.*/wiki/*
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7479]: `hmong.*` Is sufficient enough
```suggestion
hmong.*
```

**Comment by crissthiandi** [comment]
Hi, this is the url 
https://hmong.es/wiki/Isotonic_regression

**Comment by crissthiandi** [comment]
Hi! today i found that hmong have a english version, this version have the same issue with equations.
The solution is the same, i add this in the next commit

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -7476,6 +7476,14 @@ body
 
 ================================
 
+hmong.*
+wikihmong.com/*
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7480]:
```suggestion
wikihmong.com
```

**Review by Gusted — APPROVED** [review]

**Pull request merged**